packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
I still think it would be bad to recommend two separate crash tracking solutions for one app.

- Could we build a plugin for sentry itself using https://sentry.io/for/cocoa/?code=objc and https://sentry.io/for/android/ and still get stable fp for dart stacktraces?

- If not, we should at least build a plugin for registering the native platform with Sentry and recommending a path for full integration (both for native and dart crashes)? Ideally under the same app id (dsn?).